libs/output-mapping/src/Writer/Table/TableDefinition/TableDefinitionFactory.php

```diff
@@ -4,16 +4,60 @@
 
 namespace Keboola\OutputMapping\Writer\Table\TableDefinition;
 
+use Keboola\Datatype\Definition\Exasol;
+use Keboola\Datatype\Definition\Snowflake;
+use Keboola\Datatype\Definition\Synapse;
+
 class TableDefinitionFactory
 {
+    public const NATIVE_TYPE_METADATA_KEY = 'kbc.datatype.source';
```

Comment:
'kbc.datatype.backend' ? or 'kbc.backendType' ? source doesn't make much for e.g. transformations

also immediatelly document the key :)

also do we have issues for the necessary changes in snowflake ex and transformations ?

Reply:
<img width="705" alt="image" src="https://user-images.githubusercontent.com/569139/184330607-d0303f3b-09e6-4d0c-9d3a-bdca48ffbac8.png">
https://keboola.atlassian.net/browse/PS-3752
https://keboola.atlassian.net/browse/PS-3755
😄